pittgoogle/pubsub.py

```diff
@@ -322,20 +323,42 @@ class Subscription:
             Schema name of the alerts in the subscription. Passed to :class:`pittgoogle.alert.Alert` for unpacking.
             If not provided, some properties of the Alert may not be available. For a list of schema names, see
             :meth:`pittgoogle.registry.Schemas.names`.
+        attribute_filter (str, optional):
+            Specify a filter to only receive the messages whose attributes match the filter. The filter is an immutable
+            property of a subscription. After you create a subscription, you cannot update the subscription to modify
+            the filter.
+        udf (str, optional):
+            Specify a JavaScript User-Defined Function (UDF). UDFs attached to a subscription can enable a wide range
+            of use cases, including: message filtering (based on the message payload and/or attributes), simple data
+            transformations, data masking and redaction, and data format conversions.
 
     Example:
 
-        Create a subscription to Pitt-Google's 'ztf-loop' topic and pull messages:
+        Create a subscription to Pitt-Google's 'lsst-loop' topic and pull messages:
 
         .. code-block:: python
 
             # Topic that the subscription should be connected to
-            topic = pittgoogle.Topic(name="ztf-loop", projectid=pittgoogle.ProjectIds().pittgoogle)
+            topic = pittgoogle.Topic(name="lsst-loop", projectid=pittgoogle.ProjectIds().pittgoogle)
+
+            # Specify filters (Optional)
+            keepDiaObjects = "attributes:diaObject_diaObjectId"
+            filterByNPrevDetections = '''
```

Comment:
Personally, I would name these two variables `attribute_filter` and `udf` to match the kwarg names so it's clear from the start where each is destined to be applied. But whatever you prefer.